File: TFOMS/FPCS.Data/Entities/FLK.cs
using System;
using FPCS.Data.Enums;
using System.Collections.Generic;

namespace FPCS.Data.Entities
{
    public class FLK : BaseEntity
    {
        /// <summary>
        /// Id записи
        /// </summary>
        public Int64 FlkId { get; set; }

        /// <summary>
        /// Имя исходного файла
        /// </summary>
        public String BaseFileName { get; set; }

        /// <summary>
        /// Код ошибки
        /// </summary>
        public String ErrorCode { get; set; }

        /// <summary>
        /// Имя поля
        /// </summary>
        public String FieldName { get; set; }

        /// <summary>
        /// Имя базового элемента
        /// </summary>
        public String BaseEntitiyName { get; set; }

        /// <summary>
        /// Номер обращения
        /// </summary>
        public String AppealNumber { get; set; }

        /// <summary>
        /// Комментарий
        /// </summary>
        public String Comment { get; set; }

        /// <summary>
        /// Реестровый номер СМО 
        /// </summary>
        public String SmoRegNum { get; set; }
    }
}

File: TFOMS/FPCS.Web.Admin/Scripts/flk.js
flk = {

    initIndexPage: function () {
        fpcs.jqGrid.initGrid({
            gridWrapper: ".gridWrapper",
            gridSelector: "#gridTable",
            pagerSelector: "#gridPager",
            localStorageId: "FLK",
            url: "/FLK/_Index",
            rowNum: 100,
            showEditButton: false,
            showDeleteButton: false,
            sortname: 'SmoRegNum',
            jsonReader: {
                repeatitems: false,
                id: "Id"
            },
            colNames: ['Id', 'Реестровый номер СМО', 'Имя исходного файла', 'Код ошибки', 'Имя поля', 'Имя базового элемента',
                'Номер обращения', 'Комментарий'],
            colModel: [
                { name: 'Id', index: 'Id', key: true, hidden: true },
                {
                    name: 'SmoRegNum', index: 'SmoRegNum', width: 20,
                    sortable: true, resize: false
                },
                {
                    name: 'BaseFileName', index: 'BaseFileName',
                    sortable: true, width: 60
                },
                {
                    name: 'ErrorCode', index: 'ErrorCode', width: 20,
                    sortable: true, resize: false
                },
                {
                    name: 'FieldName', index: 'FieldName',
                    sortable: true, width: 30
                },
                {
                    name: 'BaseEntitiyName', index: 'BaseEntitiyName', width: 20,
                    sortable: true, resize: false
                },
                {
                    name: 'AppealNumber', index: 'AppealNumber',
                    sortable: true, width: 60
                },
                {
                    name: 'Comment', index: 'Comment', width: 120,
                    sortable: false, resize: false
                }
            ]           
        });

        fpcs.jqGrid.initGridResize();
        fpcs.jqGrid.initFilterToolbar("FLK");

        fpcs.jqGrid.initNavButtons(null, null, "");  
    },

    reloadGrid: function () {
        jQuery("#gridTable").trigger("reloadGrid")
    }

}